cmd/aks-periscope/aks-periscope.go

```diff
@@ -25,6 +25,13 @@ func main() {
 
 	clusterType := os.Getenv("CLUSTER_TYPE")
 
+	if utils.IsAzureStackCloud() {
```

Comment:
💡 Minor: Do you think it will be beneficial to move these 3/4 new line of code into separate function and all you need to add is more appropriate name for that funct: like `copyCertIfAzureStackCloud` or something more apt **Or** add couple of line of comments please for clarity.